### ISSUE TITLE
Chrome denied to access image in Google maps but Firefox worked

### ISSUE THREAD

**Issue by jackiekhuu-work** [niklasvh/html2canvas#1169]
I used html2canvas to get the image from Google Maps, everything is fine except that when I try to do some actions on Maps like move, zoom out, .... After these actions, the canvas generated the wrong image of maps. After enable logging in canvas, here is the code:

```
html2canvas(ReactDOM.findDOMNode(this.refs.fsLocation), {
          useCORS: true,
          proxy: 'http://localhost:5000',
          logging: true
        })
          .then(canvas => document.body.appendChild(canvas));
```

And then I received this error:
```
Access to Image at 'https://maps.googleapis.com/maps/api/js/StaticMapService.GetMapImage?1m2&1i…45902&2e1&3u11&4m2&1u761&2u500&5m5&1e0&5sen-US&6sus&10b1&12b1&token=114628' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:3000' is therefore not allowed access.

783ms html2canvas: Failed loading image #18 ImageContainer {src: "https://maps.googleapis.com/maps/api/js/StaticMapS…2u500&5m5&1e0&5sen-US&6sus&10b1&12b1&token=114628", image: img, tainted: null, promise: Promise} Event {isTrusted: true, type: "error", target: img, currentTarget: img, eventPhase: 2…}
modules.js?hash=24509db…:466692 785ms html2canvas: DummyImageContainer for https://maps.googleapis.com/maps/api/js/StaticMapService.GetMapImage?1m2&1i…45902&2e1&3u11&4m2&1u761&2u500&5m5&1e0&5sen-US&6sus&10b1&12b1&token=114628

933ms html2canvas: TypeError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The provided value is not of type '(CSSImageValue or HTMLImageElement or SVGImageElement or HTMLVideoElement or HTMLCanvasElement or ImageBitmap or OffscreenCanvas)'
    at CanvasRenderer.taints (modules.js?hash=24509db…:468189)
    at CanvasRenderer.drawImage (modules.js?hash=24509db…:468203)
    at CanvasRenderer.Renderer.renderImage (modules.js?hash=24509db…:468048)
    at NodeParser.<anonymous> (modules.js?hash=24509db…:467348)
    at CanvasRenderer.clip (modules.js?hash=24509db…:468213)
    at NodeParser.paintElement (modules.js?hash=24509db…:467334)
    at NodeParser.paintNode (modules.js?hash=24509db…:467320)
    at NodeParser.paint (modules.js?hash=24509db…:467296)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467102)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
    at NodeParser.asyncRenderer (modules.js?hash=24509db…:467106)
```

This is the result from Chrome:
![image](https://user-images.githubusercontent.com/12217690/28360500-ca661c04-6c9f-11e7-800f-84ddea76e709.png)

And this is the result from Firefox:
![image](https://user-images.githubusercontent.com/12217690/28360505-d1e9d952-6c9f-11e7-94ed-d2eb60043c19.png)

But It worked fine in Firefox. please help.

**Comment by jackiekhuu-work** [comment]
fixed my issue with solution from #345

**Comment by weiyaofeng** [comment]
What did you do? Ask for help
